0001-template.md

```diff
@@ -23,7 +23,7 @@ In general:
 
 * Be specific. This document is meant to share intent to your colleagues. Share what you believe you will actually do.
 * Be decisive. No maybes. Any uncertainty can be captured in the unresolved questions section at the end.
-* Provide design contex so that we can align on and commit to a technical design. 
```

Comment:
For the reviewers: changes in `001-template.md` are orthogonal, but very cosmetic, and it would be very hard to merge a separate PR because it needs a lot of approvals. So I want to add it here. It's a separate commit, so can be reverted individually if needed.